tests/occurrences/dune

```diff
@@ -0,0 +1,9 @@
+(alias
+ (name runtest)
+ (deps (:t basic.t) basic.ml)
+ (action
+   (progn
+     (setenv MERLIN %{exe:../merlin-wrapper}
+     (setenv OCAMLC %{ocamlc}
```

Comment:
That line is not needed (only merlin is used in your test, not the compiler).

Reply:
Fixed.